libraries/lib-numeric-formats/NumericConverter.h

```diff
@@ -81,11 +58,12 @@ class NUMERIC_FORMATS_API NumericConverter /* not final */
    void ParseFormatString(const TranslatableString & untranslatedFormat);
 
 public:
-   // returns true iff the format name really changed:
```

Comment:
You know "iff" is a mathematician's shorthand for "if and only if?"  When you see that word, it isn't a spelling mistake!

Reply:
But you can also write "whether" as I do sometimes, to be more precise than "if"